packages/web/src/containers/leaderboard-table-container/LeaderboardTableContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import Pagination from "@components/common/pagination/Pagination";
+import LeaderboardTableWrapper from "@components/leaderboard/leaderboard-table-wrapper/LeaderboardTableWrapper";
+import { useWindowSize } from "@hooks/common/use-window-size";
+import { useLeaders } from "@query/leaderboard";
+import MobileLeaderboardTable from "@components/leaderboard/leaderboard-table/mobile-leaderboard-table/MobileLeaderboardTable";
+import TabletLeaderboardTable from "@components/leaderboard/leaderboard-table/tablet-leaderboard-table/TabletLeaderboardTable";
+import WebLeaderboardTable from "@components/leaderboard/leaderboard-table/web-leaderboard-table/WebLeaderboardTable";
+import { useState } from "react";
+
+export default function LeaderboardTableContainer() {
+  const [page, setPage] = useState(0);
+  const movePage = (page: number) => setPage(page);
+
+  const { isMobile, isTablet, isWeb } = useWindowSize();
+  const [leadersQuery, meQuery] = useLeaders(page, 100);
+
+  return (
+    <>
+      <LeaderboardTableWrapper>
+        {isMobile && (
+          <MobileLeaderboardTable
+            me={meQuery.data?.leader}
+            leaders={leadersQuery.data!.leaders}
```

Comment:
It would be nice to resolve the warning in this syntax.
`Forbidden non-null assertion`